src/stable/build/utils/types.ts

```diff
@@ -31,15 +31,13 @@ export type Context = {
 };
 
 export type Command =
```

Comment:
Somehow create TypeScript types for the subcommands

Reply:
done